upload/admin/view/template/extension/payment/emspay.twig

```diff
@@ -59,30 +59,30 @@
                         </div>
                     </div>
 
-                    {% if emspay_module == 'emspay_klarna' %}
+                    {% if emspay_module == 'emspay_klarnapaylater' %}
                         <div class="form-group">
-                            <label class="col-sm-2 control-label" for="input-klarna-test-api-key">
-								<span data-toggle="tooltip" title="{{ info_help_klarna_test_api_key }}">
-									{{ entry_klarna_test_api_key }}
+                            <label class="col-sm-2 control-label" for="input-klarnapaylater-test-api-key">
+								<span data-toggle="tooltip" title="{{ info_help_klarnapaylater_test_api_key }}">
+									{{ entry_klarnapaylater_test_api_key }}
 								</span>
                             </label>
                             <div class="col-sm-10">
-                                <input type="text" name="payment_{{ emspay_module }}_test_api_key" id="input-klarna-test-api-key"
-                                       value="{{ _context['payment_'~emspay_module~'_test_api_key'] }}" size="50" class="form-control"
-                                       placeholder="{{ info_help_klarna_test_api_key }}"/>
+                                <input type="text" name="payment_emspay_klarnapaylater_test_api_key" id="input-klarnapaylater-test-api-key"
+                                       value="{{ _context['payment_emspay_klarnapaylater_test_api_key'] }}" size="50" class="form-control"
+                                       placeholder="{{ info_help_klarnapaylater_test_api_key }}"/>
                             </div>
                         </div>
                         <div class="form-group">
-                            <label class="col-sm-2 control-label" for="input-klarna-ip-filter">
-								<span data-toggle="tooltip" title="{{ info_help_klarna_ip_filter }}">
-									{{ entry_klarna_ip_filter }}
+                            <label class="col-sm-2 control-label" for="input-klarnapaylater-ip-filter">
+								<span data-toggle="tooltip" title="{{ info_help_klarnapaylater_ip_filter }}">
+									{{ entry_klarnapaylater_ip_filter }}
 								</span>
                             </label>
                             <div class="col-sm-10">
                                 <input type="text" name="payment_{{ emspay_module }}_ip_filter"
```

Comment:
Don't need a dynamic variable emspay_module?

Reply:
No, we don't because there is next condition above : '{% if emspay_module == 'emspay_klarnapaylater' %}' and we don't need to run controllers function for payment method name determination